common/src/main/java/com/github/teamfusion/spyglassplus/client/network/SpyglassPlusClientNetworking.java

```diff
@@ -0,0 +1,46 @@
+package com.github.teamfusion.spyglassplus.client.network;
+
+import com.github.teamfusion.spyglassplus.client.entity.LivingEntityClientAccess;
+import com.github.teamfusion.spyglassplus.network.SpyglassPlusNetworking;
+import dev.architectury.networking.NetworkManager;
+import dev.architectury.networking.NetworkManager.Side;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.nbt.NbtElement;
+import net.minecraft.nbt.NbtList;
+import net.minecraft.network.PacketByteBuf;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.github.teamfusion.spyglassplus.item.ISpyglass.*;
```

Comment:
I typically discourage use of wildcard imports, so it would be great if you could switch this to non-wildcard